docs/express.asciidoc

```diff
@@ -9,8 +9,7 @@ endif::[]
 
 Getting Elastic APM set up for your Express app is easy,
 and there are various ways you can tweak it to fit your needs.
-
-Follow the guide below and refer to the <<api,API documentation>> for all the advanced stuff.
+Follow the guide below to get started, then view the <<api,API Reference>> for the more advanced stuff.
```

Comment:
Maybe we should avoid the word "stuff" while you're in here? 😅

Reply:
😆I was trying to only fix glaring grammatical issues in this update and focus on content later, but I agree, I'll just get rid of it.

Reply:
I'm good merging this as-is. Just a comment.

Reply:
Nah I agree, let's just fix it now. Updated the branch